layouts/index.redirects

```diff
@@ -42,3 +42,13 @@ docs/v3.4.0/reporting-bugs    /docs/v3.4.0/reporting_bugs/
 /docs/v3.3/dl_build  /docs/v3.3/install/
 /docs/v3.2/dl_build  /docs/v3.2/install/
 /docs/v3.1/dl_build  /docs/v3.1/install/
+
+# Add redirects for each language except 'en'
+{{ range .Site.Languages }}
+{{- $lang := .Lang -}}
+{{- if ne $lang "en" -}}
+
+/{{ $lang }}/** /{{ $lang }}/404.html   404
```

Comment:
```suggestion
/{{ $lang }}/* /{{ $lang }}/404.html   404
```

It seems that the double asterisk (`**`) does not work as expected, no redirect to `/uk/404.html` is performed. According to netlify documentation, a single asterisk is required. (See https://docs.netlify.com/routing/redirects/redirect-options/#custom-404-page-handling)